build.gradle

```diff
@@ -58,6 +62,8 @@ subprojects {
         }
     }
 
+    project.plugins.apply('net.ltgt.errorprone')
```

Comment:
You'd want to be explicit about the version of ErrorProne you want to use rather than rely on the default `latest.release`:

```gradle
dependencies {
  errorprone 'com.google.errorprone:error_prone_core:2.0.8'
}
```

or

```gradle
configurations.errorprone {
  resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.0.8'
}
```